docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: '3'
+services:
+  server:
+    build: .
+    image: "iwantbot/server"
+    volumes:
+     - .:/app
+    ports:
+     - "8080:8080"
```

Comment:
make it listen on port `80` in the container, no need to not follow the standards when we're in a container; we have 80 available all the time